democracy/tests/test_hearing.py

```diff
@@ -6,17 +6,124 @@
+    get_hearing_detail_url, image_test_json
 )
 
 endpoint = '/v1/hearing/'
 list_endpoint = endpoint
 
 
+@pytest.fixture
+def valid_hearing_json(contact_person, default_label):
+    return {
+        "abstract": "",
+        "title": "My first hearing",
+        "id": "nD6aC5herQM3X1yi9aNQf6rGm6ZogAlC",
+        "borough": "Punavuori",
+        "n_comments": 0,
+        "published": True,
+        "labels": [{"id": default_label.id, "label": default_label.label}],
+        "open_at": "2016-09-29T11:39:12Z",
+        "close_at": "2016-09-29T11:39:12Z",
+        "created_at": "2016-10-04T10:30:38.066436Z",
+        "servicemap_url": "",
+        "sections": [
+            {
+                "type": "closure-info",
+                "commenting": 'none',
+                "published": True,
+                "title": "Section 3",
+                "abstract": "",
+                "content": "<p>Enter the introductioververn text for the hearing here.</p>",
+                "created_at": "2016-10-04T12:12:06.798574Z",
+                "created_by": None,
+                "images": [],
+                "n_comments": 0,
+                "plugin_identifier": "",
+                "plugin_data": "",
+                "type_name_singular": "sulkeutumistiedote",
+                "type_name_plural": "sulkeutumistiedotteet",
+            },
+            {
+                "commenting": 'none',
+                "published": True,
+                "title": "Section 1",
+                "abstract": "",
+                "content": "<p>Enter the introduction text for the hearing here.ve</p>",
+                "created_at": "2016-10-04T11:33:37.430091Z",
+                "created_by": None,
+                "images": [
+                    image_test_json(),
+                ],
+                "n_comments": 0,
+                "plugin_identifier": "",
+                "plugin_data": "",
+                "type_name_singular": "pääosio",
+                "type_name_plural": "pääosiot",
+                "type": "main"
+            },
+            {
+                "id": "3adn7MGkOJ8e4NlhsElxKggbfdmrSmVE",
+                "type": "part",
+                "commenting": 'none',
+                "published": True,
+                "title": "Section 2",
+                "abstract": "",
+                "content": "<p>Enter the introduction text for the hearing here.eve</p>",
+                "created_at": "2016-10-04T12:09:16.818364Z",
+                "created_by": None,
+                "images": [
+                    image_test_json(),
+                ],
+                "n_comments": 0,
+                "plugin_identifier": "",
+                "plugin_data": "",
+                "type_name_singular": "osa-alue",
+                "type_name_plural": "osa-alueet"
+            },
+        ],
+        "closed": True,
+        "organization": None,
+        "geojson": None,
+        "main_image": None,
+        "contact_persons": [{
```

Comment:
What happens if we POST a contact person that is not present in the backend? I guess the NestedPKRelatedField does not allow creating new objects and specifying their PK?  This means the other fields are redundant when posted, because this does not allow changing or creating a contact person?